shop/src/main/java/ro/msg/learning/shop/model/MainEntity.java

```diff
@@ -0,0 +1,20 @@
+package ro.msg.learning.shop.model;
+
```

Comment:
The empty lines again, tie the imports and adnotations together with only one line between them

Reply:
tied them accordingly and removed the useless empty lines